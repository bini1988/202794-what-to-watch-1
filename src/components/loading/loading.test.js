import React from "react";
import renderer from "react-test-renderer";
import Loading from "./loading";

it(`Loading correctly renders default markup`, () => {
  const tree = renderer.create(
      <Loading/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
