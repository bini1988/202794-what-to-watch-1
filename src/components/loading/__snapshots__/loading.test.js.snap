// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading correctly renders default markup 1`] = `
<div
  className="loading"
>
  <div
    className="loading__spin"
  />
</div>
`;
